packages/manager/src/features/IAM/Users/UsersTable/Users.tsx

```diff
@@ -104,20 +123,41 @@ export const UsersLanding = () => {
               User Settings
             </Typography>
           ) : (
-            <DebouncedSearchTextField
-              clearable
-              debounceTime={250}
-              errorText={searchError?.message}
-              hideLabel
-              isSearching={isFetching}
-              label="Filter"
-              onSearch={setQuery}
-              placeholder="Filter"
-              sx={{ width: 320 }}
-              value=""
-            />
+            <Grid
+              sx={(theme) => ({
+                '& > div': {
+                  width: '320px',
+                },
+                [theme.breakpoints.down('sm')]: {
+                  width: '100%',
+                },
+                [theme.breakpoints.down(XS_TO_SM_BREAKPOINT)]: {
+                  '& > div': {
+                    width: '100%',
+                  },
+                },
+              })}
+            >
+              <DebouncedSearchTextField
+                clearable
+                debounceTime={250}
+                errorText={searchError?.message}
+                hideLabel
+                isSearching={isFetching}
+                label="Filter"
+                onSearch={handleSearch}
+                placeholder="Filter"
+                value=""
+              />
+            </Grid>
```

Comment:
Instead of wrapping this in a `Grid` and applying a ton of one-off styleing, can see if we can clean this up?

Having a Grid and custom breakpoints seems a bit extra for something like this

```suggestion
            <DebouncedSearchTextField
              containerProps={{
                sx: {
                  width: { md: '320px', xs: '100%' },
                },
              }}
              clearable
              expand
              debounceTime={250}
              errorText={searchError?.message}
              hideLabel
              isSearching={isFetching}
              label="Filter"
              onSearch={handleSearch}
              placeholder="Filter"
              value=""
            />
```

Reply:
yeah, that looks good, thanks!